index.js

```diff
@@ -0,0 +1,48 @@
+const books = document.getElementById('books');
+
+const myLibrary = [];
+
+function Book(author, title, pages) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+}
+
+function addBookToLibrary(e) {
+  e.preventDefault();
+  const li = document.createElement('li');
+
+  li.className = 'collection-item';
+  const [title, author, pages] = e.target.elements;
+  const book = new Book(title.value, author.value, pages.value);
+
+  li.innerHTML = `
+  <h5>Author: <span id="author">${book.author}</span></h5>
+  <p>Title: ${book.title} </p>
+  <p>Pages: ${book.pages}</span></p>
+  <a class="delete"><i class="far fa-trash-alt"></i></a>
+  <label>
+  <input type="checkbox" />
+  <span>Read</span>
+</label>
+  `;
+  console.log(li);
```

Comment:
- [ ] Kindly remove the console.log. This is because it kills performance and exposes sensitive application data. ⬇️ 
![console_log_requirement](https://user-images.githubusercontent.com/32361963/115720655-f0bf9f80-a385-11eb-8929-d6c2b1f49f77.png)